packages/pwa-kit-react-sdk/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 ## v4.0.0-extensibility-preview.5 (May 06, 2025)
+- Fix data-fetching regression with SSR rendering [#2785](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2785)
```

Comment:
Similar comment as above, but it would be nice to angle this in a positive light since "regression" almost implies we unintentionally broke something and released it and fixed it. Since we didn't release it we might be able to say "Account for rending components that use reacts useInsertionEffect hook on the server"